src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -13,7 +13,7 @@ public static class NuGetConstants
         public static readonly string V2LegacyFeedUrl = "https://go.microsoft.com/fwlink/?LinkID=230477";
 
         public static readonly string V1FeedUrl = "https://go.microsoft.com/fwlink/?LinkID=206669";
-        public static readonly string DefaultGalleryServerUrl = "https://www.nuget.org";
+        public static readonly string DefaultGalleryServerUrl = "https://api.nuget.org/v3/index.json";
```

Comment:
Looks like only push and setapikey use this, www.nuget.org is not a endpoint, I don't know why we put this URL here, anyway, will double check this. @emgarten